target/linux/ramips/base-files/etc/init.d/enablemodem

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh /etc/rc.common
+. /lib/ramips.sh
+
+START=99
+
+start() {
+	local board=$(ramips_board_name)
+	if [ $board = "mr200" ]; then
+		adb shell chmod +x /WEBSERVER/www/cgi-bin/*
+		adb shell httpd -h /WEBSERVER/www/ &
+		echo "2357 000d" > /sys/bus/usb-serial/drivers/option1/new_id
```

Comment:
@HRomie if You first run adb server then add new id to option driver You can connect to adb shell and use port tty to comunicate with modem.
Modem have 7 interfaces:
0 - rndis
1 - rndis
2 - option
3 - usbfs (adb interface)
4 - option
5 - option
6 - usbstorage

When You first use echo command then interface 3 use option driver.